.gitignore

```diff
@@ -53,6 +53,7 @@ systest/
 *.swp
 clients/src/generated
 clients/src/generated-test
+metadata/src/generated
```

Comment:
I'm confused because this is already in .gitignore.  I think you need to rebase...